config/locales/fr.yml

```diff
@@ -2,6 +2,17 @@ fr:
   date:
     formats:
       short: "%d-%m-%Y"
+  daily_question_added_successfully!: Nouvelle question ajoutée avec succès !
+  daily_question_updated_successfully!: Nouvelle question mise à jour avec succès !
+  daily_question_destroyed_successfully!: Nouvelle question supprimée avec succès !
+  question_added_successfully!: Questions ajoutées avec succès !
+  question_updated_successfully!: Question mise à jour avec succès !
+  question_destroyed_successfully!: Question supprimée avec succès !
+  documentation_added_successfully!: Documentation ajoutée avec succès !
+  documentation_updated_successfully!: Documentation mise à jour avec succès !
+  documentation_destroyed_successfully!: Documentation supprimée avec succès !
+  user_updated_successfully!: Utilisateur mis à jour avec succès !
```

Comment:
I think you could factorise those notices, since they'll probably be the same for all CRUD actions on all models.
Here is what you could do:

```yml
fr:
  controllers:
    crud:
      success:
        create: "%{model.human} ajouté(e) avec succès !"
        update: "%{model.human} #%{id} modifiée(e) avec succès."
        destroy: "%{model.human} #%{id} supprimée."
      errors:
        create: "Erreur : %{model.human} n'a pas été créé(e)."
        update: "Erreur : %{model.human} #%{id} n'a pas pu être modifié(e)."
        destroy: "Erreur : impossible de supprimer %{model.human} #%{id}."
```

And in views, you could then use:

```ruby
redirect_to questions_path, notice: t("controllers.crud.success.create", model: @question.class)
```